src/components/SingleCategory.js

```diff
@@ -18,8 +18,10 @@ const SingleCategory = (props) => {
                 <img className="review-image" src={item.review_img_url} alt={item.title}/>
                 <p>{item.title}</p>
                 <p class="designer">{item.designer}</p>
```

Comment:
And if you extracted that logic as suggested in my previous comment - you could re-use it here and your code would be so much DRY-er!